tests/import_osm/test_create_hierarchy.py

```diff
@@ -19,6 +19,7 @@ def test_osm_polygon_parent_id_get_set_if_covered(session, schema, tables):
                 name="Some Polygon with missing parent",
                 admin_level=16,
                 country_code='ch',
+                type='city',
```

Comment:
Nice! :+1: 

Just to add on, we were discussing once if the type could null. I thought about it, it shouldn't be, because the mapping_value becomes the 'type' ([link](https://github.com/OSMNames/OSMNames/blob/master/data/import/mapping.yml#L9)). Only features with a mapping_value as specified in the yaml file will be imported, hence they will all have a type. :)